### PR TITLE
add simp_le

### PR DESCRIPTION
**Pull request by phunehehe**
@fmap should these be arranged somehow differently to make it easier to clean up after we move to 16.03?

### CODE REVIEW

**Comment by fmap** [comment]
If there're problems, I'll figure them out when they come up, merge at will.

File: sdk.nix [hunk 1]
```diff
@@ -106,35 +106,7 @@ rec {
...
-      s3cmd shadow sshfsFuse stdenv strace strongswan sysstat systemd
-      tcpdump tmux tree tzdata
-      unzip utillinux
-      vim
-      wget which
-      xz
-    ;
-
-    inherit (pkgs) bundler;
+  exports = pkgs // rec {
```
Comment (ip1981) [inline sdk.nix:109]: :+1:

Comment (ip1981) [inline sdk.nix:109]: Next step is automatic pick-up / override as in

``` nix
{ pkgs, lib, ... }:

let
  all = lib.attrNames (
    lib.filterAttrs
    ( n: _: n != "default.nix" && ! lib.hasPrefix "." n )
    (builtins.readDir ./.)
  );

  localPackages = map (f:
    { ${lib.removeSuffix ".nix" f} = pkgs.callPackage (./. + "/${f}") {}; }
  ) all;

in {
  nixpkgs.config.packageOverrides =
    (lib.foldl (a: b: a//b) {} localPackages);
}
```

Comment (phunehehe) [inline sdk.nix:109]: Thanks Igor, we'll eventually get there :)

**Pull request merged**